### PR TITLE
Add CHANGELOG file for Releases

### PR DESCRIPTION
**Pull request by cesnietor**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Added MinIO Console Help Menu
+- Updated UI Menu components
+
+Bug Fix:
+
+- Disable Upload button on Object Browser if user not allowed.
+- Fixed security vulnerability for `lestrrat-go/jwx`
+- Fixed security vulnerability for `fast-xml-parser`
+- Fixed bug on subtpaths for Object Browser
```
Comment (harshavardhana) [inline CHANGELOG.md:17]:
```suggestion
- Fixed bug on sub-paths for Object Browser
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Features:
+
+- Added MinIO Console Help Menu
+- Updated UI Menu components
+
+Bug Fix:
+
+- Disable Upload button on Object Browser if user not allowed.
+- Fixed security vulnerability for `lestrrat-go/jwx`
+- Fixed security vulnerability for `fast-xml-parser`
```
Comment (harshavardhana) [inline CHANGELOG.md:16]: Combine them

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- Fixed bug on subtpaths for Object Browser
+- Fixed minor bugs on Tooltips
```
Comment (harshavardhana) [inline CHANGELOG.md:18]: Not needed

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- Reduce number of calls to `/session` api to improve performance.
```
Comment (harshavardhana) [inline CHANGELOG.md:19]:
```suggestion
- Reduce the number of calls to `/session` API endpoint to improve performance.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+- Rolled back previous change for Share File feature to no longer ask for Service Account access keys.
```
Comment (harshavardhana) [inline CHANGELOG.md:20]:
```suggestion
- Rolled back the previous change for the Share File feature to no longer ask for Service Account access keys.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+## Release v0.30.0
+
+Features:
+
+- Added MinIO Console Help Menu
+- Updated UI Menu components
+
+Bug Fix:
+
+- Disable Upload button on Object Browser if user not allowed.
```
Comment (harshavardhana) [inline CHANGELOG.md:14]:
```suggestion
- Disable the Upload button on Object Browser if the user is not allowed.
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (cesnietor) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 1 above)
Comment (cesnietor) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (cesnietor) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 6 above)
Comment (cesnietor) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 5 above)
Comment (cesnietor) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 4 above)
Comment (cesnietor) [reply CHANGELOG.md]: done

**Review by donatello — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**